### ISSUE TITLE
Chemist King Discount Pharmacy (Australian pharmacy/chemist)

### ISSUE THREAD

**Issue by davidhicks** [alltheplaces/alltheplaces#4584]
### Brand name

Chemist King Discount Pharmacy

### Wikidata ID

Q63367667

### Store finder url(s)

https://www.chemistking.com.au/

(note: sitemap does not allow location pages to be separated hence the main landing page needs parsing to extract individual store pages from a drop-down menu)

(most HTML classes and identifiers are randomly generated and are probably unstable, hence XPath/CSS selectors could be a bit more difficult to choose)

**Comment by Cj-Malone** [comment]
Does this only have 9 locations? If so, it's probably not worth any ones time, especially since they are doing weird things.

**Comment by davidhicks** [comment]
Yes only 9 locations per https://www.chemistking.com.au/stores (just found this URL lists all 9 stores but parsing of the 9 individual pages is still needed). You're right that hardly anyone is going to miss this spider if it's too much effort.
